zigpy_zigate/zigbee/application.py

```diff
@@ -26,6 +31,8 @@ def __init__(self, api, database_file=None):
 
     async def startup(self, auto_form=False):
         """Perform a complete application startup"""
+        self._api = ZiGate.new(self._config[CONF_DEVICE], self)
```

Comment:
await is missing
should be `self._api = await ZiGate.new(self._config[CONF_DEVICE], self)`